cla_public/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -1512,23 +1544,23 @@ msgid "These cookies need to be on to give you the best experience using the ser
 msgstr "Mae angen i’r cwcis hyn fod ar waith er mwyn i chi gael y profiad gorau wrth ddefnyddio’r gwasanaeth."
 
 #: cla_public/templates/_cookie-consent-form.html:37
-msgid "Read more about cookies on 'Check if you can get legal aid'"
+msgid "Read more about cookies on ‘Check if you can get legal aid’"
 msgstr "Darllenwch fwy am ddefnyddio cwcis ar ‘Gwirio os ydych yn gymwys i gael cymorth cyfreithiol’"
 
 #: cla_public/templates/_cookie-consent-form.html:42
-msgid "Cookies used to improve 'Check you can get legal aid'"
+msgid "Cookies used to improve ‘Check if you can get legal aid’"
 msgstr "Cwcis sy’n cael eu defnyddio i wella ‘Gwirio os ydych yn gymwys i gael cymorth cyfreithiol’"
```

Comment:
I wrote this in a previous comment, but used wrong references...

- I just wanted to know, if adding "if" to the to make it 'Check if you can get legal aid' will warrant an extra Welsh word being added to the translation or not.

Reply:
That's a good question.  

I don't think so.  All instances, regardless of "if", are translated as "Gwirio os ydych yn gymwys i gael cymorth cyfreithiol", so we're good.

The only variation is on line 1402, where it is translated as "Gwirio os y gallwch chi gael cymorth cyfreithiol".  Something for another ticket perhaps.